components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -378,16 +499,13 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
    real(r8), pointer :: snow_sh(:)          ! snow from shallow convection              [m/s]
 
    ! stratiform precipitation variables
-   real(r8), pointer :: prec_pcw(:)         ! total precip from prognostic cloud scheme [m/s]
-   real(r8), pointer :: snow_pcw(:)         ! snow from prognostic cloud scheme         [m/s]
    real(r8), pointer :: prec_sed(:)         ! total precip from cloud sedimentation     [m/s]
-   real(r8), pointer :: snow_sed(:)         ! snow from cloud ice sedimentation         [m/s]
-   real(r8), pointer :: snow_str(:)         ! snow from stratiform cloud                [m/s]
+
+   real(r8), pointer :: ttend_dp(:,:)       ! Convective heating for gravity wave drag
 
    integer lchnk                                   ! chunk identifier
    integer ncol                                    ! number of atmospheric columns
    integer nstep                                   ! time steps
-   real(r8) crm_run_time                           ! length of CRM integration
```

Comment:
I agree to replace crm_run_time with ztodt. Do you mind adding 'length of CRM integration' to the description of 'ztodt' ? It makes the meaning of ztodt clearer, imo.

Reply:
will do.